toplevel/dune

```diff
@@ -0,0 +1,84 @@
+(library
+ (name ocamltoplevel)
+ (wrapped false)
+ (flags (:standard -nostdlib -w -9))
+ (libraries stdlib ocamlcommon ocamlbytecomp)
+ (modules genprintval toploop trace topdirs topmain))
+
+(executable
+ (name topstart)
+ (modes byte)
+ (flags (:standard -nostdlib))
+ (libraries ocamlbytecomp ocamlcommon ocamltoplevel runtime stdlib)
+ (modules topstart))
+
+(executable
+ (name expunge)
+ (modes byte)
+ (flags (:standard -nostdlib))
+ (libraries ocamlbytecomp ocamlcommon runtime stdlib)
+ (modules expunge))
+
+(rule
+ (targets ocaml.byte)
+ (action (run %{ocaml_where}/expunge topstart.exe %{targets}
+                    ; FIXME: inlined $(STDLIB_MODULES) ... minus Labels ones ...
```

Comment:
Discussed with @trefis, we have some code in utop to avoid writing the list of modules explicitly:

https://github.com/ocaml-community/utop/blob/master/src/top/expunge/expunge.ml

Called as follow: https://github.com/ocaml-community/utop/blob/master/src/top/expunge/expunge.ml

We could modify `expunge` to do that directly.

Reply:
I'll try to do that in a follow-up PR.